Objects/ComingObjects/ComingObject.gd

```diff
@@ -2,12 +2,14 @@ extends Sprite
 class_name ComingObject
 signal timeout
 
+export var wait_time = 2
```

Comment:
Cool, but make it float, i.e. in this way: `export(float) var wait_time = 2.0` ;)